.github/actions/process_recipe/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM pangeo/pangeo-notebook:latest
```

Comment:
Question: don't we want to pin a specific version here? This image gets updated ~every two weeks. Breaking changes could creep in because of this.

Reply:
Agreed.  We are going to maintain image definitions tagged with pinned versions of Prefect and pangeo-forge in https://github.com/pangeo-forge/docker-images.  Something like pangeo-forge/pangeo-forge-worker:14.3_0.2.0.  We'll push images to a public registry for use here in this workflow and to individual cloud provider registries so that workers don't experience throttling and avoid data transfer costs when pulling images.